applications/rag/workloads.tfvars

```diff
@@ -31,70 +30,42 @@ create_gcs_bucket    = true
 # To verify, check that `gcloud storage buckets describe gs://<bucketname>` returns a 404.
 gcs_bucket = "rag-data-<username>"
 
+# Ensure the instance name is unique to your project.
 cloudsql_instance        = "pgvector-instance"
```

Comment:
Can we generate this somehow from the cluster name?

Reply:
will do in followup PR